src/control/lib/hardware/pciutils/bindings.go

```diff
@@ -0,0 +1,120 @@
+//
+// (C) Copyright 2024 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package pciutils
+
+import (
+	"os"
+	"unsafe"
+
+	"github.com/daos-stack/daos/src/control/lib/hardware"
+	"github.com/pkg/errors"
+)
+
+/*
+#cgo LDFLAGS: -lpci
+
+#include <stdlib.h>
+#include <pci/pci.h>
+*/
+import "C"
+
+type api struct{}
+
+func (api *api) Cleanup() {}
```

Comment:
If we don't expect `Cleanup` to be used, no need to define it in the interface, in my opinion.

Reply:
this was added by the initial implementation. removed